rust/integration-tests/sdk_expected_db_output_files2/imported_testnet_txns/fungible_asset_processor/8/coin_supply.json

```diff
@@ -1,10 +1,10 @@
 [
   {
-    "transaction_version": 21,
+    "transaction_version": 8,
```

Comment:
Maybe we even stub out transaction_version, transaction_timestamp, and transaction_epoch, since these are not relevant to the test. Can add as a TODO

Reply:
yeah wanted to hear your opinion on these, wasn't sure if they would really care about those.

Reply:
I think your point of using the txn_version to debug a failed test is valid, so we should probably keep for now.